code/controllers/subsystems/news.dm

```diff
@@ -0,0 +1,154 @@
+/var/datum/controller/subsystem/news/SSnews
+
+/datum/controller/subsystem/news
+	name = "News"
+	flags = SS_NO_FIRE
+	var/list/datum/feed_channel/network_channels = list()
+	var/datum/feed_message/wanted_issue
+
+/datum/controller/subsystem/news/Recover()
+	src.network_channels = SSnews.network_channels
+	src.wanted_issue = SSnews.wanted_issue
+
+/datum/controller/subsystem/news/New()
+	NEW_SS_GLOBAL(SSnews)
+
+/datum/controller/subsystem/news/Initialize(timeofday)
+	CreateFeedChannel("Station Announcements", "SS13", 1, 1, "New Station Announcement Available")
+	if(config.sql_enabled)
+		load_from_sql()
+	..()
+
+/datum/controller/subsystem/news/proc/load_from_sql()
+	if(!establish_db_connection(dbcon))
+		log_debug("SSnews: SQL ERROR - Failed to connect.")
+		return
+	var/DBQuery/channel_query = dbcon.NewQuery("SELECT id, name, author, locked, is_admin_channel, announcement FROM ss13_news_channels WHERE deleted_at IS NULL ORDER BY name ASC")
+	channel_query.Execute()
+	while(channel_query.NextRow())
+		CHECK_TICK
+		try
+			CreateFeedChannel(
+				channel_query.item[2],
+				channel_query.item[3],
+				text2num(channel_query.item[4]),
+				text2num(channel_query.item[5]),
+				channel_query.item[6])
+		catch(var/exception/ec)
+			log_debug("SSnews: Error when loading channel: [ec]")
+		CHECK_TICK
```

Comment:
This is unnecessary here, all things considered. Maybe consider moving onto line 43 instead?